game/views.py

```
@@ -61,17 +61,113 @@ def _sanitize_move_history(move_history):
+            'white_name': _clean_player_name(payload.get('white_name'), 'White'),
+            'black_name': _clean_player_name(payload.get('black_name'), 'Black'),
+            'difficulty': payload.get('difficulty') or 'medium',
+        }
+    return {
+        'white_name': 'White',
+        'black_name': 'Black',
+        'difficulty': 'medium',
+    }
+
+
+def _state_from_request_data(request, data=None, allow_new=False):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_state_from_request_data`

(ANN202)